CODING_GUIDELINES.md

```diff
@@ -31,7 +31,60 @@ Security:
     + code must be always deterministic
 + Thread safety. If some functionality is not thread-safe, or uses something that is not thread-safe, then clearly indicate the risk on each level.
 
-## Testing
+## Acceptance tests
```

Comment:
nice addition, can you add some example links or recommended ways to do these. Right now the Regen team understand this but it will be hard for others to onboard into this.

Reply:
There are 2 links with more details:  Martin F, and openclassrooms.com. I will add a link to an acceptance tests we have at Regen.